release-notes/3.0/preview/api-diff/preview9/Asp.Net/3.0-preview9.md

```diff
@@ -0,0 +1,24 @@
+# API Difference netcoreapp3.0-preview8 vs netcoreapp3.0-preview9
+
+API listing follows standard diff formatting. Lines preceded by a '+' are
```

Comment:
Let's be consistent with the file naming convention. Preview 8 had ASP capitalized, where we now have Asp camel-cased.

Please change one or the other (I'd be fine with changing the diff fore preview 8) to follow this style.